example/run_hypershard.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Maybe link to this in the readme or alternatively turn this into a wiki page?

Reply:
Added here https://github.com/dropbox/hypershard-android/pull/10/files#diff-04c6e90faac2675aa89e2176d2eec7d8R34